packages/browser/src/index.ts

```diff
@@ -197,7 +198,7 @@ export default class LogtoClient {
   }
 
   public async fetchUserInfo(): Promise<UserInfoResponse> {
-    const { userinfoEndpoint } = await this.getOidcConfig();
+    const { userinfoEndpoint } = await this.getOidcConfigOnce();
```

Comment:
it's ok to remove `Once` since the caller doesn't care?

Reply:
Renamed. PTAL.